docs/data/rs_data_en.md

```diff
@@ -0,0 +1,107 @@
+
+![band](../images/band.jpg)
+
+The electromagnetic spectrum is so wide that it is impractical to use a single sensor to collect information at all wavelengths at once. In practice, different sensors give priority to collecting information from different wavelengths of the spectrum. Each part of the spectrum captured and classified by the sensor is classified as an information strip. The tape varies in size and can be compiled into different types of composite images, each emphasizing different physical properties. At the same time, most remote sensing images are 16-bit images, different from the traditional 8-bit images, which can represent finer spectral information.
+
+## 4 Classification of Remote Sensing Images
+
+Remote sensing image has the characteristics of wide coverage area, large number of bands and rich sources, and its classification is also very diverse. For example, remote sensing image can be divided into low resolution remote sensing image, medium resolution remote sensing image and high resolution remote sensing image according to spatial resolution. According to the number of bands, it can be divided into multi-spectral image, hyperspectral image, panchromatic image and other types. This document is intended to provide a quick guide for developers who do not have a background in remote sensing. Therefore, only a few common types of remote sensing images are described.
+
+### 4.1 RGB Image
+
+RGB images are similar to common natural images in daily life. The features displayed in RGB images are also in line with human visual common sense (for example, trees are green, cement is gray, etc.), and the three channels represent red, green and blue respectively. The figure below shows an RGB remote sensing image:
+
+![rgb](../images/rgb.jpg)
+
+Since most of the current CV task processing processes are designed based on natural images, remote sensing data sets of RGB type are widely used in CV field.
+
+### 4.2 MSI/HSI Image
+
+MSI (Multispectral Image) and HSI (Hyperspectral Image) usually consist of several to hundreds of bands, The two are distinguished by different spectral resolution (*spectral resolution refers to the value of a specific wavelength range in the electromagnetic spectrum that can be recorded by the sensor; the wider the wavelength range, the lower the spectral resolution*). Usually the spectral resolution in the order of 1/10 of the wavelength is called multispectral. Compared with HSI, MSI has less band number, wider band and higher spatial resolution. However, HSI has more bands, narrower bands and higher spectral resolution.
```

Comment:
`Compared with HSI, MSI has less band number, wider band and higher spatial resolution. However, HSI has more bands, narrower bands and higher spectral resolution.`
->
`MSI has fewer bands, wider band, and higher spatial resolution, while HSI has more bands, narrower bands, and higher spectral resolution.`